src/mod_tools.hpp

```diff
@@ -1070,17 +1070,25 @@ class MonsterStatCustomManager
 		outputPath.append(fileName.c_str());
 
 
-		FILE* fp = fopen(outputPath.c_str(), "wb");
+		File* fp = FileIO::open(outputPath.c_str(), "wb");
 		if ( !fp )
 		{
 			return;
 		}
 		char buf[65536];
-		rapidjson::FileWriteStream os(fp, buf, sizeof(buf));
-		rapidjson::PrettyWriter<rapidjson::FileWriteStream> writer(os);
+		int count = fp->read(buf, sizeof(buf[0]), sizeof(buf));
+		buf[count] = '\0';
+		rapidjson::StringStream os(buf);
+		rapidjson::PrettyWriter<rapidjson::StringStream> writer(os);
 		d.Accept(writer);
+		for (int c = 0;;++c) {
+			if (buf[c] == '\0') {
```

Comment:
Wouldn't this check be equivalent if done in the for loop?
```for (int c = 0; buf[c] != '\0'; ++c)```

Reply:
it would :) changed